source/PlayerInfoPanel.cpp

```diff
@@ -559,29 +596,57 @@ void PlayerInfoPanel::DrawFleet(const Rectangle &bounds)
 	Color bright = *GameData::Colors().Get("bright");
 	Color elsewhere = *GameData::Colors().Get("dim");
 	Color dead(.4f, 0.f, 0.f, 0.f);
-	
+	Color special(0.5f, 0.5f, 0.1f, 0.f);
+
+	// Table columns and their starting x positions
+	vector <tuple <string, int, Table::Align,
```

Comment:
Not sure if tuple was the right choice here it can get a little unreadable further down. Thoughts?

Reply:
Yes, readability is a bit low there.

The compare-functions are only needed after the relevant zone gets clicked, so you could remove them here and move them into a separate look-up table that gets used after a sort-button gets clicked. (The lookup table could be "column-name" -> sort-function)